src/main/java/racing/domain/Cars.java

```diff
@@ -0,0 +1,48 @@
+package racing.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import racing.view.OutputView;
+
+public class Cars {
+	private List<Car> cars;
+
+	public Cars(List<String> input) {
+		cars = input.stream().map(Car::new).collect(Collectors.toList());
```

Comment:
협업할 때 다른 개발자들도 읽기 좋게 엔터로 구분해주면 좋겠어요
```suggestion
		cars = input.stream()
				.map(Car::new)
				.collect(Collectors.toList());
```

Reply:
여러곳에서 발견되어 수정하였습니다.